x-pack/plugins/security_solution/public/timelines/components/timeline/tabs/session/use_session_view.tsx

```diff
@@ -273,18 +271,8 @@ export const useSessionView = ({ scopeId, height }: { scopeId: string; height?:
     [globalFullScreen, scopeId, timelineFullScreen]
   );
 
-  const sourcererScope = useMemo(() => {
```

Comment:
I'm always a bit scared of removing code that's been here for a few years, but if I understand correctly, this was probably needed before when we had the old flyout, but I'm actually not sure why it would be different now...
The original change was done in 2022 when the decouple from TGrid happened... 😨

Reply:
yeah.. it might have been overlooked when the flyout was switched to expandable flyout. using the events index pattern would always throw an error when opening an alert flyout.